tests/network-tests/src/fixtures/membership/UpdateProfileHappyCaseFixture.ts

```diff
@@ -66,6 +68,12 @@ export class UpdateProfileHappyCaseFixture extends BaseQueryNodeFixture {
       externalResources: isSet(this.newValues.externalResources)
         ? this.newValues.externalResources || null
         : this.oldValues.externalResources,
+      isVerifiedValidator: isSet(this.newValues.isVerifiedValidator)
+        ? this.newValues.isVerifiedValidator
+        : this.oldValues.isVerifiedValidator,
+      validatorAccount: isSet(this.newValues.validatorAccount)
+        ? this.newValues.validatorAccount
```

Comment:
We expect the `validatorAccount` to be `null` (i.e removed) when an empty string was passed:
```suggestion
        ? this.newValues.validatorAccount || null
```